runtime/loongarch64.S

```diff
@@ -0,0 +1,827 @@
+/**************************************************************************/
```

Comment:
You might want to sync this file with its `riscv64.S` counterparts, at least with the following changes:
- Optimal polling in C code based on masking (`3f9b1225eda0f21f5c63bd68c6700137be4ce888`)
- Store both ends of the stack chain in continuations (`f80990e2e27a8bb7fd5f38d9da2e37ee301a2ba4`)
- Function sections (`2a16b0e17f7e9d4fcc80a2929c16cdcc87f5b797`)
- Various cleanups (`63de4f1475a036e7b68801402fa8bf0f0dbf4494`, `ebc5e1d242941357f4f00483c358d3aeb1c8f176`)

Reply:
Thank you so much for your patience in providing me with the reference commit id, it helps a lot!